templates/repo/release/list.tmpl

```diff
@@ -99,7 +99,7 @@
 									<a href="{{AppSubUrl}}/{{.Publisher.Name}}">{{.Publisher.Name}}</a>
 								</span>
 								{{ end }}
-								<span class="ahead"><a href="{{$.RepoLink}}/compare/{{.TagName | EscapePound}}...{{.Target}}">{{$.i18n.Tr "repo.release.ahead.commits" .NumCommitsBehind | Str2html}}</a> {{$.i18n.Tr "repo.release.ahead.target" $.DefaultBranch}}</span>
+								<span class="ahead"><a href="{{$.RepoLink}}/compare/{{.TagName | EscapePound}}{{if .Target}}...{{.Target}}{{end}}">{{$.i18n.Tr "repo.release.ahead.commits" .NumCommitsBehind | Str2html}}</a> {{$.i18n.Tr "repo.release.ahead.target" $.DefaultBranch}}</span>
```

Comment:
Does 1.15 have this open compare url?

Reply:
Good question .... it seems not ....

https://github.com/go-gitea/gitea/blob/37abfcaf8a012076901a8847f273488e5b038031/routers/web/repo/compare.go#L156-L196

Reply:
So I wonder to know why that target is empty. 🤔

Reply:
I added a fallback to defaultbranch.